test/aws-lambda/features/promise-rejection.feature

```diff
@@ -42,7 +42,7 @@ Scenario Outline: unhandled promise rejections are not reported when autoDetectE
     When I invoke the "<lambda>" lambda in "features/fixtures/simple-app" with the "events/<type>/promise-rejection.json" event
     Then the lambda response "errorMessage" equals "Error: yikes"
     And the lambda response "errorType" equals "Runtime.UnhandledPromiseRejection"
-    And the lambda response "trace" is an array with 6 elements
+    And the lambda response "trace" is an array with 5 elements
```

Comment:
The change here is that the line `"    at process.EventEmitter.emit (domain.js:483:12)"` is no longer present. This frame was being added due to the mere presence of `require('domain')` in the contextualize plugin, which is part of the node notifier and therefore also instantiated in aws lambda.